.gitmodules

```diff
@@ -7,4 +7,4 @@
 	branch = releases/tag/v0.5.2-fix
 [submodule "deps/selene"]
 	path = deps/selene
-url=https://github.com/adam4813/Selene
\ No newline at end of file
+	url = https://github.com/adam4813/selene
```

Comment:
This will most likely merge conflict as I changed the source back to the upstream repo

Reply:
Ah whoops!